MANIFEST.in

```diff
@@ -1,5 +1,5 @@
 graft pybamm
-include CITATION.cff
+include CITATION.cff pybamm/input/comsol_results/*.json
```

Comment:
Might be better to use something like `global-include <pattern to include>`, i.e., `global-include comsol_*.json` in a new line